doc/element_globals.tex

```diff
@@ -1,17 +1,17 @@
-The \texttt{GLOBALS} block holds singular data model instances, and is not associated 
-with a particular VOTable \texttt{TABLE} element.
+The \texttt{GLOBALS} block holds model instances or collections of model instances 
+that are not connected with any table column. 
 
-The contained instances have a global scope, and may be
-referenced by other instances anywhere in the mapping block.  \texttt{INSTANCE} attributes
-within the \texttt{GLOBALS} scope may only refer to VOTable \texttt{PARAMS} or contain
-explicit values, they MUST NOT refer to VOTable \texttt{FIELDS}.  \\ 
-Note: This rule is not enforced via the MIVOT XSD schema which is restricted to the control of the mapping block only.
+The instances contained in \texttt{GLOBALS} have a global scope, and may be
+referenced by other instances anywhere in the mapping block.  
+They must only contain
+explicit values, they must not refer to VOTable \texttt{FIELDS}.  \\ 
```

Comment:
I'm not sure if you define "explicit" somewhere.  If not, I think you should go back to "references to PARAMs or literals" or something like that.  Or just say "references must not go to FIELDs (even transitively) in GLOBALS" -- but of course you'd save the whole trouble if you dropped GLOBALS, which I think would be a serious improvement.

Reply:
text rephrased. 
Note: GLOBALS is a placeholder for meta-data that are missing in the VOTable (filter, dataset metadata...)